jangjuseong/controllers/likeController.js

```diff
@@ -0,0 +1,11 @@
+const likeService = require('../services/likeService');
+
+const postLike = async (req, res) => {
```

Comment:
함수명을 수정해주면 좋을 것 같습니다!
```suggestion
const addLike = async (req, res) => {

// 또는

const createLike = async (req, res) => {
```